build/check.js

```diff
@@ -109,6 +109,34 @@ gulp.task('check-javascript-format', function() {
       .pipe(gulpClangFormat.checkFormat('file', undefined, {verbose: true, fail: true}));
 });
 
+/**
+ * Checks whether project's Go files are formatted by goimorts.
+ */
+gulp.task('check-go-format', function(doneFn) {
```

Comment:
What is the purpose of adding the command to list the files that don't match the output of goimports? Why wouldn't you just run 'gulp format-go'?

Reply:
'gulp format-go' runs 'goimports -w src/app/backend', and goimports overwrites sources.
But goimports removes necessary import lines sometimes. So we should not run `goimorts -w src/app/backend` implicitly.

Reply:
Hmm, ok. goimports removing import lines seems like something goimports shouldn't do. The Go compiler will produce the required errors if there are unnecessary imports.

Anyway, thanks for the explanation.